stopcovid/status/users.py

```diff
@@ -126,17 +139,22 @@ def get_user(self, user_id: uuid.UUID) -> Optional[User]:
     def get_drill_status(self, user_id: uuid.UUID, drill_slug: str) -> Optional[DrillStatus]:
         result = self.engine.execute(
             select([drill_statuses]).where(
-                drill_statuses.c.user_id == func.uuid(str(user_id))
-                and drill_statuses.c.drill_slug == drill_slug
+                and_(
+                    drill_statuses.c.user_id == func.uuid(str(user_id)),
+                    drill_statuses.c.drill_slug == drill_slug,
+                )
             )
         )
         row = result.fetchone()
         if row is None:
             return None
+        drill_instance_id = (
+            uuid.UUID(row["drill_instance_id"]) if row["drill_instance_id"] else None
```

Comment:
Is `None` an actual case here?

Reply:
We populate a drill status row for each of the slugs once a user is validated. Before the user starts a drill, a drill instance ID doesn't exist. I'll add a comment.